test/test_framework.py

```diff
@@ -786,6 +786,7 @@ class SomeObject(Object):
         obj.state.foo = 41
         obj.state.foo = 42
         obj.state.bar = "s"
+        obj.state.baz = 4.2
```

Comment:
We should also test `obj.state.bing = True`

Reply:
```suggestion
        obj.state.baz = 4.2
        obj.state.bing = True
```